### PR TITLE
Sync content from docusaurus to gastby

### PR DESCRIPTION
**Pull request by chartrandf**
## Context

Avoid content duplication between beta.wazo-platform.org and live website (Gatsby).

## Summary of change

Do not allow files in `content/blog` (Gastby) but only sync them from Docusaurus. Until the new website is ready, this will make us maintaining one place for contents.

- Delete / ignore all files inside `content/blog`
- Scripts that sync content and make them valid for Gastby website

### CODE REVIEW

File: website/release-notes/sprint-review-2402.md
Comment (Romej) [inline website/release-notes/sprint-review-2402.md:1]: new empty file ?

File: website/release-notes/sprint-review-2403.md
Comment (Romej) [inline website/release-notes/sprint-review-2403.md:1]: new empty file ?

**Review by Romej — COMMENTED** [review]
Doing manual tests

File: scripts/beta-sync.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Goal: Temporary script that copy beta content to old website until migration will be done
```
Comment (Romej) [inline scripts/beta-sync.sh:1]:
```suggestion
#!/bin/bash
# Goal: Temporary script that copy beta content to old website until migration will be done
```

File: scripts/beta-sync.sh [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+
+# 1. Sync release note & blog posts
+echo "🔃 Sync release note & blog posts from docusaurus\n"
+rm -r content/blog || true
+mkdir content/blog
+cp website/release-notes/*.md website/blog/*.md content/blog/
+
+sed -i '' 's/tags: \[\(.*\)\]/tags: \1/g' content/blog/*.md # tags support
+sed -i '' 's/..\/static/https:\/\/beta.wazo-platform.org\//g' content/blog/*.md # image source
```
Comment (Romej) [inline scripts/beta-sync.sh:11]:
```suggestion
if [[ "$OSTYPE" == "darwin"* ]]; then
    # macOS
    sed_cmd="sed -i ''"
else
    # Linux
    sed_cmd="sed -i"
fi

# Modify tags format
$sed_cmd 's/tags: \[\(.*\)\]/tags: \1/g' content/blog/*.md
# Update image source URLs
$sed_cmd 's|../static|https://beta.wazo-platform.org/|g' content/blog/*.md
```

**Review by Romej — CHANGES_REQUESTED** [review]
Suggested change so that it works on Linux too

**Review by Romej — COMMENTED** [review]
`src/component/blog/article.js` seems to be missing the `author` : 

image

File: website/release-notes/sprint-review-2402.md
Comment (chartrandf) [reply website/release-notes/sprint-review-2402.md:1]: Permissions change, was having execute mode.

**Review by Romej — APPROVED** [review]

**Pull request merged**